pulp_ansible/app/tasks/collections.py

```diff
@@ -1,12 +1,14 @@
 import asyncio
 from gettext import gettext as _
-import json
 import logging
 import math
-import tarfile
 
 from django.db import transaction
 
+# TODO(awcrosby) rename in galaxy-importer to support this:
```

Comment:
Can we use `from galaxy_importer.importer import process_collection` and remove these comments? Or keep `as process_collection` and remove the comments? Either is ok w/ me.

Reply:
```bash
In [2]: from galaxy_importer.collection import process_collection                                                                                                                             
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-2-cc42a578c3e1> in <module>
----> 1 from galaxy_importer.collection import process_collection

ImportError: cannot import name 'process_collection' from 'galaxy_importer.collection' (/usr/local/lib/pulp/lib64/python3.7/site-packages/galaxy_importer/collection.py)

In [3]: from galaxy_importer.collection import import_collection                                                                                                                              

In [4]:    
```
So I just removed the comments